runtime/frequency/src/lib.rs

```diff
@@ -747,6 +747,7 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 	type MaxMembers = CouncilMaxMembers;
 	type DefaultVote = pallet_collective::PrimeDefaultVote;
 	type WeightInfo = weights::pallet_collective_council::SubstrateWeight<Runtime>;
+	type SetMembersOrigin = EnsureRoot<Self::AccountId>;
```

Comment:
Is this what the default was before?

Reply:
SetMembersOrigin is `Origin allowed to set collective members`
New for 9.39, I couldn't easily track down what the default was prior.

Reply:
We should definitely test governance with this upgrade.  e.g.  `proposeToBeProvider()`